build-system/tasks/vendor-configs.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const gulp = require('gulp');
+const gulpif = require('gulp-if');
+const gulpWatch = require('gulp-watch');
+const jsonlint = require('gulp-jsonlint');
+const jsonminify = require('gulp-jsonminify');
+const rename = require('gulp-rename');
+const {endBuildStep, printNobuildHelp, toPromise} = require('./helpers');
+
+/**
+ * Entry point for 'gulp vendor-configs'
+ * @return {!Promise}
+ */
+async function vendorConfigs() {
+  printNobuildHelp();
+  return compileVendorConfigs();
+}
+
+/**
+ * Compile all the vendor configs and drop in the dist folder
+ * @param {Object=} opt_options
+ * @return {!Promise}
+ */
+function compileVendorConfigs(opt_options) {
+  const options = opt_options || {};
+
+  const srcPath = 'extensions/amp-analytics/0.1/vendors/*.json';
+  const destPath = 'dist/v0/analytics-vendors/';
```

Comment:
Would this be a good place to put all vendor configs? @erwinmombay

Reply:
are there any other main binaries that will be using it?

Reply:
No, only `amp-analytics` will be using these files